server/ctrlsubsonic/handlers_playlist.go

```diff
@@ -238,10 +235,14 @@ func playlistRender(c *Controller, params params.Params, playlistID string, play
 			continue
 		}
 		trch.TranscodeMeta = transcodeMeta
-		resp.List = append(resp.List, trch)
+		if withItems {
+			resp.List = append(resp.List, trch)
+		}
 	}
 
-	resp.SongCount = len(resp.List)
+	if withItems {
+		resp.SongCount = len(resp.List)
```

Comment:
Your commit log mentions duration, but you are only filling out the SongCount field, is this intentional?